library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/focals/CirclePromptFocal.java

```diff
@@ -125,14 +137,17 @@ public void draw(@NonNull Canvas canvas)
         if (mDrawRipple)
         {
             final int oldAlpha = mPaint.getAlpha();
+            final int oldColor = mPaint.getColor();
+            mPaint.setColor(Color.WHITE);
```

Comment:
The ripple should be the same colour as the the rest of the focal, add another Paint for the ripple and add a check to set it to white for a transparent focal in setColour